src/components/Slider/Slider.js

```diff
@@ -0,0 +1,19 @@
+function Slider({ symptom, description, fieldsetIndex, points, handleChange }) {
```

Comment:
To bym nazwała SleepAndItchSlider
```suggestion
function SleepAndItchSlider({ symptom, description, fieldsetIndex, points, handleChange }) {
```

Reply:
done